src/ripple/shamap/impl/SHAMapDelta.cpp

```diff
@@ -58,14 +58,14 @@ bool SHAMap::walkBranch (SHAMapAbstractNode* node,
             // This is a leaf node, process its item
             auto item = static_cast<SHAMapTreeNode*>(node)->peekItem();
 
-            if (emptyBranch || (item->getTag () != otherMapItem->getTag ()))
+            if (emptyBranch || (item->key() != otherMapItem->key()))
             {
                 // unmatched
                 if (isFirstMap)
-                    differences.insert (std::make_pair (item->getTag (),
+                    differences.insert (std::make_pair (item->key(),
```

Comment:
Gosh, this code is repetitive.  It's way beyond the scope of the code review, but if something was obvious to you... (my guess is that we're actually just creating a `DeltaRef` and then the insert could be a single statement).

Totally not mandatory, just a note since I never saw this code before...